setup/README.md

```diff
@@ -7,24 +7,24 @@ A guide to setting up your Raspberry Pi
 ### Essential (for general use)
 
 - [SD card](../installation/sd-cards.md)
-    - We recommend a minimum of 8GB class 4 or class 10 microSD card. To save time, you can get a card that is pre-installed with [NOOBS](../installation/noobs.md) or [Raspberry Pi OS](../installation/installing-images/README.md), although setting up your own card is easy.
+    - We recommend a minimum of 8GB micro SD card, and using [Raspberry Pi Imager](https://www.raspberrypi.org/software/) to install an operating system onto it.
 - [Display and connectivity cable](monitor-connection.md)
     - Any HDMI/DVI monitor or TV should work as a display for the Pi. For best results, use a display with HDMI input; other types of connection for older devices are also available.
 - Keyboard and mouse
     - Any standard USB keyboard and mouse will work with your Raspberry Pi.
     - Wireless keyboards and mice will work if already paired.
     - For keyboard layout configuration options see [raspi-config](../configuration/raspi-config.md).
 - [Power supply](../hardware/raspberrypi/power/README.md)
-    - The Pi is powered by a USB Micro [models pre 4B] or USB Type-C [model 4B] power supply (like most standard mobile phone chargers).
-    - You need a good-quality power supply that can supply at least 3A at 5V for the Model 4B, 2A at 5V for the Model 3B and 3B+, or 700mA at 5V for the earlier, lower-powered Pi models. We recommend using the [official Raspberry Pi power supply](https://www.raspberrypi.org/products/raspberry-pi-universal-power-supply/), which is designed specifically for Raspberry Pi.
+    - The Pi is powered via a micro USB connector, except the Pi 4B and Pi 400 which use a USB type C connector.
+    - You need a good quality power supply that can supply at least 3A at 5V for the Pi 4B, 2A at 5V for the Pi 3B and 3B+, or 700mA at 5V for the earlier, lower-powered Pi models. We recommend using the [official Raspberry Pi power supply](https://www.raspberrypi.org/products/raspberry-pi-universal-power-supply/), which is designed specifically for Raspberry Pi.
```

Comment:
There should probably also be a link to https://www.raspberrypi.org/products/type-c-power-supply/ as well?

Maybe it's worth removing the specific current-requirements here, and just refer the reader to `../hardware/raspberrypi/power/README.md` instead?

Reply:
> There should probably also be a link to https://www.raspberrypi.org/products/type-c-power-supply/ as well?
> 
> Maybe it's worth removing the specific current-requirements here, and just refer the reader to `../hardware/raspberrypi/power/README.md` instead?

I was trying to avoid making too many edits to this page, as there's probably not much point spending too much time on it. However, since you've provided good solutions to the two problems I had already spotted I will make the suggested changes.